langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -99,6 +111,9 @@ public isolated function sum(int... ns) returns int = @java:Method {
 
 # Returns the maximum of one or more int values.
 #
+# ```ballerina
+# int:max(50, 20, 30, 70, 65) ⇒ 70
```

Comment:
We can add an example with the method call syntax here. Please add wherever applicable.